code-challenges.test.js

```diff
@@ -15,19 +20,56 @@
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
-
-
-// b) Create the function that makes the test pass.
-
+// --->TEST 
+ def("original")("original1"), () => {
+      it("takes in an array, removes the first item from the array and shuffles the remaining content."), () => {
+          expect(original("colors1").toEqual(["yellow", "blue", "pink", "green"]))
+      }
+  }
```

Comment:
Syntax is still a little off on your test.  You should be using the describe method, not def(def is when you are creating a method in Ruby).  I highly recommend using a template and refactoring it each time as the sytax is pretty tricky for tests.  Here is an example: `describe("myFunction", () => { it("description of what function will do", () => { expect(myFunction(argument)).toEqual(expected output) }) })`